src/Shortcodes/ImageShortcodeProvider.php

```diff
@@ -85,6 +88,17 @@ public static function handle_shortcode($args, $content, $parser, $shortcode, $e
             }
         }
 
+        // Image lazy loading - must be globally enabled (is enabled by default)
```

Comment:
This method is already too big as is. New logic should be in a new `private` method. If time allows split the existing method into smaller private method as well.

Reply:
Updated